cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/datastore/Datastore.java

```diff
@@ -30,7 +29,7 @@
  * <p>The Datastore can be viewed as a pool of heterogeneous JSON documents. One
  * datastore can hold many different types of document, unlike tables within a
  * relational model. The datastore provides hooks, particularly in the
- * {@link DatastoreExtended} interface, which allow for various querying models
+ * {@link DatastoreImpl} class, which allow for various querying models
```

Comment:
Since DatastoreExtended has been retired, the 'extra' functionality not available through the Datastore interface now has to be accessed through DatastoreImpl.

That's what this clumsy sentence (which I didn't originally write) is trying to say - should I re-write it?